src/components/core/popoverMenu/popoverItem.vue

```diff
@@ -32,8 +32,8 @@
 		<!-- If item.input is set instead, an put will be used -->
 		<span v-else-if="item.input" class="menuitem">
 			<input :id="item.key" :type="item.input" :class="item.input"
-				v-model="item.model" @change="item.action">
-			<label :for="item.key">{{ item.text }}</label>
+				v-model="item.model">
+			<label :for="item.key" @click="item.action">{{ item.text }}</label>
```

Comment:
CanI also keep the @click though? Because the checkbox doesn't recognise the @change because we're creating it with the ::before

Reply:
Hum, the change should work. The label is still linked to the input. My guess is that we have an event prevent somewhere :)
Let me check :)

Reply:
Indeed, the change is not forwarded to the input.
We need to do keep both then. :)

Reply:
ok good to know thank you :)

Reply:
Aaaand, I was wrong :p 
It's because you did not use a key, therefore the input and label were not connected together :( 
I added an automated key for this purpose. I'll push it right away! :tada: